tableaudocumentapi/multilookup_dict.py

```diff
@@ -40,6 +46,14 @@ def __setitem__(self, key, value):
 
         dict.__setitem__(self, key, value)
 
+    def get(self, key, default_value=AttributeError):
+        try:
+            return self[key]
+        except AttributeError:
+            if default_value != AttributeError:
```

Comment:
I don't actually want to test to see if it's an instance of the class, but the class itself.  The reason for this is I need a sentinel that is highly unlikely to be passed in to determine if a default value was passed in or not.  Unless you know of a better way to determine that?  I suppose I could create a tuple instead, and test for that specific tuple.